### DISCUSSION TITLE
İbax cüzdan kodlarımı nerede görebilirim? Where can I see my ibax wallet codes?

### DISCUSSION THREAD

**Discussion by ergun818** [IBAX-io/go-ibax#946] [Soru Cevap - Türkçe] [answered]
İBAX platformuna Twitter üzerinden erişim sağlamıştım daha önce cüzdan oluşturdum fakat kodları yedeklemedim.Sürekli Twitter ile giriş sağlıyorum.İbax cüzdanıma ait Seed'leri nerede görebilirim tekrar?

I had access to the IBAX platform via Twitter, I created a wallet before, but I did not back up the codes. I always log in with Twitter. Where can I see the Seeds of my Ibax wallet again?

**Comment by zukenberger** [comment]
kanka kaydetmediysen en baştan açacaksın ibax walleti. Youtube "Kripto Tugay" diye kanal var oradan ibax videolarını bir izle o iyi anlatıyor.

**Comment by Yusufknk** [comment, marked as answer]
Eğer cüzdanınız tarayıcınızda kayıtlı ise weaver.ibax.io adresinden cüzdanınıza giriş yapabilirsiniz. Private keylerini almadıysanız ve cüzdana erişim sağlayamıyorsanız, eski cüzdan kelimelerini ve şifresini yeni cüzdan oluştur kısmından girerek eski cüzdanınıza erişim sağlayabilirsiniz.
